source/guide_dendrite.rst

```diff
@@ -49,22 +49,22 @@ Check out a stable version (replace ``v0.13.5`` with the latest version):
 .. code-block:: console
 
   [isabell@stardust ~]$ cd ~/dendrite/
-  [isabell@stardust dendrite]$ git checkout v0.13.5
+  [isabell@stardust ~/dendrite]$ git checkout v0.13.5
```

Comment:
nope. Default bash config will only show the current folder name, not a full path here

Reply:
Right, I removed this change.